packages/store/src/data-access/filesystem.ts

```diff
@@ -0,0 +1,233 @@
+import path from 'path';
+import fs, { readFileSync, writeFileSync, mkdirSync, FSWatcher, existsSync } from 'fs';
+import { plural } from 'pluralize';
+import { DataAccess, DataAccessConstructor } from '.';
+import { Logger } from '@sap-ux/logger';
+import { errorInstance, getFioriToolsDirectory } from '../utils';
+import { Entity } from '..';
+import { ServiceOptions } from '../types';
+import os from 'os';
+
+export const basedir = ({ baseDirectory }: { baseDirectory?: string } = {}): string => {
+    if (!baseDirectory) {
+        return getFioriToolsDirectory();
+    } else if (path.isAbsolute(baseDirectory)) {
+        return baseDirectory;
+    } else {
+        return path.join(os.homedir(), baseDirectory);
+    }
+};
+
+export const FilesystemStore: DataAccessConstructor<object> = class implements DataAccess<object> {
```

Comment:
jsdoc

Reply:
Fixed in https://github.com/SAP/open-ux-tools/pull/237/commits/e37e1de9ea46e33d3a9e87c4febfd4187ff5186a